concepts/50 React Components/45 Optimize Performance/10 Keep Elements out of the Main Function.md

```diff
@@ -0,0 +1,46 @@
+If an object literal does not require an access to the variables, you can extract if from the function body.
+
+	<!-- tab: App.js -->// Incorrect:
```

Comment:
Do not use tab symbols because different text editors interpret them differently.